src/System.Private.ServiceModel/tools/test/Bridge/Bridge/Program.cs

```diff
@@ -16,6 +16,15 @@ namespace Bridge
 {
     internal class Program
     {
+        // Exit codes returned by this executable
+        private const int Bridge_ExitCode_Success = 0;
+        private const int Bridge_ExitCode_Failure = -1;
+        private const int Bridge_ExitCode_AlreadyRunning = 1;
+        private const int Bridge_ExitCode_Started = 2;
+        private const int Bridge_ExitCode_Stopped = 3;
+        private const int Bridge_ExitCode_NotRunning = 4;
+        private const int Bridge_ExitCode_NotStopped = 5;
```

Comment:
We don't really need the exit code 2, 3, 4 and 5. For the purpose of this PR, all we want to know is if the bridge is already running (exit code 1). For all other commands (eg. start, stop etc.), an exit code of either success (0) or failure (-1) will be sufficient.